pkgs/top-level/all-packages.nix

```diff
@@ -3316,6 +3316,10 @@ in
 
   usbsdmux = callPackage ../development/tools/misc/usbsdmux { };
 
+  usbview = callPackage ../tools/misc/usbview {
+    inherit (gnome3) gtk;
```

Comment:
This is an alias, `gtk3` should be used instead.

Reply:
Fixed.